ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java

```diff
@@ -62,12 +61,26 @@ public void addPatientAppointmentTest() throws Exception {
 
     @After
     public void tearDown() throws Exception {
+    	removeWeekends();
```

Comment:
please use the same indention or spacing as is found in the this same file, consider the next line for-example